webcommon/javascript2.editor/src/org/netbeans/modules/javascript2/editor/embedding/JsEmbeddingProvider.java

```diff
@@ -172,159 +160,6 @@ protected interface Translator {
         public List<Embedding> translate(Snapshot snapshot);
     }
 
-    private static final class JspTranslator implements Translator {
```

Comment:
Just out of curiosity, what was the purpose of these removed Translators?

Reply:
My understanding is, that the translators are actually embedding providers, that used various techniques to extract JS out of different source file formats. The removed translators are actually not referenced, for example the xhtml processor was move to the `html.editor` module.